fix: improved flush() docs in to standalone Android session replay (#260)
amplitude/amplitude-dev-center: docs/session-replay/sdks/standalone-android.md

1. Create a  `val sessionReplay = SessionReplay()` object to begin collecting replays. Pass the API key, session identifier, and device identifier.
2. When the session identifier changes, pass the new value to Amplitude with `sessionReplay.setSessionId`.
3. Collect Session Replay properties to send with other event properties with `sessionReplay.getSessionReplayProperties`

> 4. Send the session replay data to Amplitude by calling `sessionReplay.flush`. Always call flush before the app is exited or sent to the background. For longer sessions, flush should be called periodically to prevent high memory usage (alpha).

Reviewer: ```suggestion
4. Call `sessionReplay.flush` to send session replay data to Amplitude. Always call `flush` before exiting the app or sending it to the background. For longer sessions, call flush frequently to prevent high memory use (alpha).
```
